esignet-service/src/main/java/io/mosip/esignet/controllers/ClientManagementController.java

```diff
@@ -52,7 +59,8 @@ public ResponseWrapper<ClientDetailResponse> updateClient(@Valid @PathVariable("
         try {
             response.setResponse(clientManagementService.updateOIDCClient(clientId, requestWrapper.getRequest()));
         } catch (IdPException ex) {
-            auditWrapper.logAudit(Action.OIDC_CLIENT_UPDATE, ActionStatus.ERROR, AuditHelper.buildAuditDto(clientId), ex);
+            auditWrapper.logAudit(((String) SecurityContextHolder.getContext().getAuthentication().getPrincipal()),
```

Comment:
Use AuditHelper.getClaimValue, this could throw NPE.

Reply:
Changed it to AuditHelper.getClaimValue